docs/openapi_generator/pyopenapi/utility.py

```diff
@@ -114,3 +116,64 @@ def write_html(self, f: TextIO, pretty_print: bool = False) -> None:
         )
 
         f.write(html)
+
+def get_all_api_protocols() -> List[Type[Any]]:
+    """Get all API protocol classes from the APIs directory."""
+    protocols = []
+    # Get the root directory of the project (where llama_stack is located)
+    root_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), "../../.."))
+    apis_dir = os.path.join(root_dir, "llama_stack", "apis")
+
+    for root, _, files in os.walk(apis_dir):
```

Comment:
can I suggest using a more direct approach to get all API protocols using 

https://github.com/meta-llama/llama-stack/blob/main/llama_stack/distribution/resolver.py#L61

instead of walking the directory?